extendeddatacrossword.go

```diff
@@ -274,9 +274,16 @@ func (eds *ExtendedDataSquare) rebuildShares(
 func (eds *ExtendedDataSquare) verifyAgainstRowRoots(
 	rowRoots [][]byte,
 	r uint,
-	shares [][]byte,
+	oldShares [][]byte,
+	rebuiltIndex int,
```

Comment:
Separating out leads to a lot of code duplication because `noShareInsertion` needs to be propagated all the way inside to the last function call. Which is why I included `noShareInsertion` in the first place